profiles/templates/profiles/view_profile.html

```diff
@@ -15,4 +15,72 @@ <h3>Created Events:</h3>
 {% else %}
   <p>No events created yet.</p>
 {% endif %}
+
+<div class="col-md-4 my-3">
+  {% if user.is_authenticated %}
+      {% if user != user_profile.user %}
+          <!-- The user is not the creator of the event -->
+          {% if join_status == PENDING %}
+              <!-- Button to withdraw request -->
+              <form action="{% url 'profiles:toggle-friend-request' user_profile.id %}" method="post">
+                  {% csrf_token %}
+                  <button type="submit" class="btn btn-outline-primary btn-block join-button" id="withdraw-to-join" onclick="return confirm('Are you sure you want to withdraw your friend request?');">Remove Friend Request</button>
+              </form>   
+          {% elif join_status == WITHDRAWN or join_status == None or join_status == REJECTED or join_status == REMOVED %}
+              <!-- Button to join again or create a new EventJoin object-->
+              <form action="{% url 'profiles:toggle-friend-request' user_profile.id %}" method="post">
+                  {% csrf_token %}
+                  <button type="submit" class="btn btn-primary btn-block join-button" id="request-to-join" onclick="return confirm('Ready to send a request to join?');">Send Friend Request</button>
+              </form>
+          {% elif join_status == APPROVED %}
+          <!-- show that the request is approved-->
+          <!-- <div class="btn btn-primary btn-block join-button" id="approved-fake">Friend Requests</div>   -->
+          {% endif %}
+          <div class="approved-join-list sticky-note-container my-4">
+              <h3>Friends</h3>
+              <div class="join-entry">{{ join.user }}</div>
+              {% for join in approved_join %}
+                  <div class="join-entry">
+                      {{ join.user }}</div>
+              {% endfor %}
+          </div>
```

Comment:
Since adding friends is a mutual relation. The user who gets approved should also have a button to remove the friend he/she previously requested.